src/components/GFERequestPanel.js

```diff
@@ -281,6 +287,10 @@ const OrganizationSelect = (organizations, organizationSelected, label, id, hand
     <Select required labelId={label} id={id} value={organizationSelected} onOpen={handleOpen} onChange={handleSelect} style={{ backgroundColor: "#FFFFFF" }}>
         {
             organizations ? (organizations.map((org) => {
+                //filters to only receive type "provider"
+                if (org.resource.type[0].coding[0].code !== "prov") {
+                    return
+                }
```

Comment:
For the submitting provider, we should allow all the orgs except insurance company (type === "pay") as we should allow intermediary org to submit the GEF as well.